content/fall_recruitment.md

```diff
@@ -0,0 +1,171 @@
+For CLEF 2025 alone, we had 43 published authors! 
+By joining us, you'll gain valuable experience, build a network of like-minded individuals, and have the opportunity to participate in competitive research. You can read more about our achievements on our [Impact](impact) page. 
+
+The interest group takes place in the fall semester and our goal is to learn about data science and research competitions and strategies for success. 
+It will culminate in forming competition teams for CLEF 2026, a conference that systematically evaluates information access systems primarily through experimentation on shared tasks. 
+All DS@GT members are welcome, regardless of your level of experience. 
+
+# Suggested Prequisites
+
+- You should have prior programming experience. 
+- You should be comfortable writing code in Python using the SciPy stack (numpy, pandas, matplotlib). 
+- No prior competition experience is necessary.
+
+# Interest Group Format
+The group will have two sections that meet weekly in the evening during Central European Time (CEST) and US Pacific Time (PST). 
+Meetings are 45 minutes in length and will vary in content. 
+All meetings will be held and recorded on Zoom. 
+The rough general schedule for the group is as follows:
+
+**August** - Recruitment and Kickoff
+
+**September** - Intro to Applied Research
+
+**October** - Applied Research Continued
+
+**November** - Cloud Computing Basics on PACE and Team Formation
+
+Members can read, present, and discuss research, and write an exploratory data analysis (EDA). 
+Expected time commitment for the Fall is 2-3 hours a week, with 1 hour for meetings and 1-2 hours preparing for the next week. 
+Participation in the interest group will weigh heavily into Spring competition team placement. 
+Participation in the interest group and returning members who published with us before will be prioritized for team placement. 
+This has worked well for us in the past and we have not encountered an issue where there are more interested people than the amount of teams available. 
+If more help is needed, we will also have a separate interview process at the end of the semester for broader recruitment.
+
+I hope you find this exciting and attend our first meeting and kickoff when the Fall semester starts! 
+At the first meeting, we'll introduce the club, discuss our goals and activities, and answer any questions you may have. 
+Please [RSVP](https://docs.google.com/forms/d/e/1FAIpQLSfODo9AxoE0exzTFT2115nYWbYw4dWfDK3jvnDvKoKH-3LkbA/viewform?usp=dialog) and we’ll remind you when dates are solidified at the end of the first week. 
+Feel free to share this document and join us in #applied-research-competitions by joining us on the DS@GT Slack: https://linktr.ee/datasciencegt 
+
+# Points of Contact: 
+
+Murilo Gustineli murilogustineli@gatech.edu
+
+Ritesh Mehta rmehta307@gatech.edu
+
+# FAQ (make it more fall semester focused?)
```

Comment:
Maybe we should move general FAQ into a separate page? That, or I'd try to prune as many of the questions as possible to make this less overwhelming.